.github/workflows/pypi.yaml

```diff
@@ -1,4 +1,5 @@
----
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Too bad for consistency that this file uses 4 spaces indent instead of 2 like other files...

Reply:
I guess this could be "fixed" in a followup PR.

Reply:
Definitely.